Pipfile

```diff
@@ -0,0 +1,41 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+alembic = "==1.5.2"
+appdirs = "==1.4.3"
+attrs = "==19.3.0"
+black = "==19.10b0"
+click = "==7.0"
+GeoAlchemy2 = "==0.6.3"
+Mako = "==1.1.0"
+MarkupSafe = "==1.1.1"
+more-itertools = "==7.2.0"
+pathspec = "==0.7.0"
+psycopg2 = "==2.8.4"
+python-dateutil = "==2.8.0"
+python-editor = "==1.0.4"
+regex = "==2020.1.8"
+SQLAlchemy = "==1.4.15"
+toml = "==0.10.0"
+typed-ast = "==1.4.1"
+
+[dev-packages]
+alembic-verify = "==0.1.4"
+atomicwrites = "==1.3.0"
+importlib-metadata = "==0.23"
+pytest = "==3.10.1"
+pytest-describe = "==0.11.1"
+pytest-mock = "==3.2.0"
+py = "==1.8.0"
+scramp = "==1.1.0"
+"testing.common.database" = "==2.0.3"
+"testing.postgresql" = "==1.3.0"
+pg8000 = "==1.13.2"
+pluggy = "==0.13.0"
+six = "==1.12.0"
+sqlalchemy-diff = "==0.1.5"
+SQLAlchemy-Utils = "==0.37.3"
+zipp = "==0.6.0"
```

Comment:
I would consider putting a comment in here (if the text format allows it) noting why there is no `python_version`.